sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldText.cs

```diff
@@ -6,6 +6,8 @@
 namespace Azure.AI.FormRecognizer.Models
 {
     /// <summary>
+    /// Represents either a key or a value in the key-value pair that constitutes
```

Comment:
For this, let's go with something more like:

A form content element representing text that is part of a form field.  This includes the location of the text in the form and a collection of the content elements that make up the text.